coderd/audit/request.go

```diff
@@ -94,6 +99,10 @@ func ResourceID[T Auditable](tgt T) uuid.UUID {
 		return typed.Group.ID
 	case database.APIKey:
 		return typed.UserID
+	case database.License:
+		// TODO: return a valid ID here when the below ticket is completed:
+		// https://github.com/coder/coder/pull/6012
+		return uuid.New()
```

Comment:
idk that we can merge this up with this line the way it is.

Reply:
is there a reason we can't just use the numerical id?

Reply:
@coadler Do you mean the the `License.ID` field? That is type `int32` - if we used that that, how would we get the `AuditLog` model to accept it, given it only takes UUIDs?
Another issue: for some reason, `License.ID` changes. Like if I delete 'license 4' in the CLI, the `License.ID` will change from 4  to 0. IDK why we're doing this.

Reply:
@coadler fixed with the merge of #6012!